Source/WebCore/rendering/FlexibleBoxAlgorithm.h

```diff
@@ -60,7 +60,7 @@ class FlexItem {
 
     LayoutUnit constrainSizeByMinMax(const LayoutUnit) const;
 
-    RenderBox& box;
+    CheckedPtr<RenderBox> box;
```

Comment:
Why not a CheckedRef ?

Reply:
No reason not to actually. Changed and makes the code change smaller